packages/front-end/components/ProjectBadges.tsx

```diff
@@ -14,14 +13,14 @@ export interface Props {
     | "member"
     | "team"
     | "fact table"
+    | "feature"
```

Comment:
This component was meant for resources with an array of projects.  Feature only has a single project.  This is mostly fine, except we want the empty state to show "None" instead of "All" for features

Reply:
Fixed